test/obb/events/game_events_test.clj

```diff
@@ -21,29 +21,20 @@
   (let [system (-> (game-events-system) component/start)
         game-events (:game-events system)
         game-info (game-info/create)
-        subscriber-ch (async/chan 1)
+        game-ch (async/chan 1)
         subscriber-all-ch (async/chan 1)
         test-event :new-game
-        expected-message {:event-name test-event
-                          :game-id (game-info/id game-info)
+        expected-message {:game-id (game-info/id game-info)
                           :data game-info}]
```

Comment:
I was wondering if it would make sense to define other format for the data on the events. `game-id` and `data` both relate to the game, so why not create only one key named `game`? I think it would be the best interface for the consumers and it is not too generic like `data`. At Talkdesk I was always advocating that we should treat events like a second delivery mechanism. In a way, the processing of an event is very similar to handling a request. All sorts of business logic can be executed in reaction to an event. This means that the data we send in an event can be viewed as the list of arguments for that piece of logic. We should send only what is needed and we should carefully choose the interface.

On a side note, I think we will also want to add some kind of metadata to the events. That can be definitely left for later, but I would recommend something like:

```clj
{:game {...} ; game information
 :_meta {:timestamp ... ; unix timestamp
         :uuid ...}; event universal identifier
```

Just wanted to leave it here so we can discuss.